packages/x-scheduler/src/primitives/use-event-occurrences-with-timeline-position/useEventOccurrencesWithTimelinePosition.ts

```diff
@@ -0,0 +1,219 @@
+import * as React from 'react';
+import {
+  CalendarEventOccurrence,
+  CalendarEventOccurrenceWithTimePosition,
+  SchedulerValidDate,
+} from '../models';
+import { useAdapter } from '../utils/adapter/useAdapter';
+import { Adapter } from '../utils/adapter/types';
+
+// A small buffer to consider events that are very close but not really overlapping as overlapping.
+const COLLISION_BUFFER_MINUTES = 5;
```

Comment:
I'm not 100% sure we need this 🤔 
In this case, I would expect the second event to start on column 1 
<img width="284" height="158" alt="image" src="https://github.com/user-attachments/assets/3242c1c6-29b4-4b50-8e9f-27008b24a1a5" />

Reply:
Bryntum doesn't have this behavior
I can keep the logic but set the bugger value to 0 for now :ok_hand: